### PR TITLE
D3ASIM-2910: deprecate old blockchain dependencies (WIP)

### PR DESCRIPTION
**Pull request by lana-shanghai**

### CODE REVIEW

**Review by hannesdiedrich — DISMISSED** [review]
LGTM

File: requirements/base.in [hunk 1]
```diff
@@ -9,9 +9,7 @@ pendulum==2.1.2
 pip-tools
 plotly
 # TODO: Deactivating solidity compiler due to bug
```
Comment (spyrostz) [inline requirements/base.in:11]: Please also remove this, thanks !

**Review by spyrostz — DISMISSED** [review]
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**